src/main/webapp/breakfast_page.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <title>Breakfast</title>
```

Comment:
I'd suggest taking a look at using URL parameters and a JavaScript function to change the HTML and "content" div, similar to how you've done it before in the portfolio walkthroughs: https://www.sitepoint.com/get-url-parameters-with-javascript/

In that case your nav bar would redirect to links like "my-app.appspot.com/index.html?category=brunch" or "my-app.appspot.com/index.html?category=breakfast".

`index.html` would have something like `<body onload="populateCategory()">` and your JS file would have something like:
```
func populateCategory() {
     const urlParams = new URLSearchParams(window.location.search);
     var category = urlParams.get('category');
     document.title = category;
     // modify div named "content" similarly
}
```

Not blocking on this recommendation because it's relatively complex and we want forward progress, but I highly recommend looking into this. It will save you a lot of copy-paste work when you want to change up your nav bar or other visual elements.